### PR TITLE
Added feature to play an instant mix from selected artists.

### PR DESCRIPTION
**Pull request by faizzzzz**

### CODE REVIEW

File: lib/components/MusicScreen/ArtistItemListTile.dart [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                leading: Icon(Icons.star),
+                title: Text("Add Favourite"),
+              ),
+            ),
+            _jellyfinApiData.selectedMixArtistsIds.contains(mutableItem.id) ?
+            const PopupMenuItem<ArtistListTileMenuItems>(
+              value: ArtistListTileMenuItems.RemoveFromMixList,
+              child: ListTile(
+                leading: Icon(Icons.explore_off),
+                title: Text("Remove From Mix List"),
```
Comment (faizzzzz) [inline lib/components/MusicScreen/ArtistItemListTile.dart:95]: please suggest a more user friendly message for the add and remove options.

**Comment by UnicornsOnLSD** [comment]
I'll have a look at this tomorrow :)

File: lib/components/MusicScreen/ArtistItemListTile.dart (hunk 1 above)
Comment (UnicornsOnLSD) [reply lib/components/MusicScreen/ArtistItemListTile.dart:97]: "Add to mix" and "Remove from mix" should be good enough

**Comment by faizzzzz** [comment]
Sure, no worries.

**Comment by faizzzzz** [comment]
Not sure why the checks are failing.

**Comment by UnicornsOnLSD** [comment]
Checks are failing because I've moved user stuff into a new singleton (FinampUserHelper). You'll need to fetch upstream and use the new singleton (`_finampUserHelper.currentUser` instead of `_jellyfinApiData.currentUser`).

**Comment by faizzzzz** [comment]
Pulled in master and fixed some errors. The Android build succeeded but iOS is failing in `flutter pub get`

**Comment by UnicornsOnLSD** [comment]
Code looks good, I'm assuming the iOS build is just some weird macOS quirk. Merged :)

**Comment by faizzzzz** [comment]
ok, sounds good. 
If this UI works, I can add it for the "Albums" tab as well.

**Pull request merged**